### ISSUE TITLE
Feature Request: Monitor Discord Bots

### ISSUE THREAD

**Issue by tankcr** [appu1232/Discord-Selfbot#318]
We have a server to discord bot, but the plugin doesn't pick up anything it posts from the server. Just requesting this feature be added :)

**Comment by LyricLy** [comment]
...what?

**Comment by LyricLy** [comment]
Ah, the keyword logger? Yes, it ignores bots, which is by design.

**Comment by LyricLy** [comment]
Not currently, no. How does this not work for those scenarios?

**Comment by tankcr** [comment]
ok consider, you are running a game server, you want to monitor in game chat for swearing through the discord bot, say for instance.

**Comment by LyricLy** [comment]
Incorporate that into the bot itself?

**Comment by PulzHF** [comment]
I see what he asking for- for example- if a game server chat is relayed through a Discord bot (webhooks or bot messages) and you want alerts when people say certain words.
I'd find that mighty useful.

**Comment by netflixandbill** [comment]
If I’m understanding the issue correctly, you want to monitor for keywords or phrases. Just make a private channel in any server you have the proper elevated permissions for and use the notify/log cog. Set that up to be specifically looking in the server(s) you need to and then make a word list. Whenever those keywords are said the webhook you create will show the message containing the word and a few previous messages above it for context, who said it and in which channel it was said.

**Comment by LyricLy** [comment]
No, they specifically want to log bots, which are ignored by that method.

**Comment by tankcr** [comment]
exactly!

**Comment by dasaesch** [comment]
same here. we need notifications for messages created from bots. This is not working at the current version.
